### PR TITLE
[lighthouse] make doc titles match audit titles

### PR DESCRIPTION
**Pull request by kaycebasques**
Just minor updates so that the reference doc titles match the audit titles pretty much verbatim

### CODE REVIEW

File: src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
-description: Reference documentation for the "HTML Has a Viewport Meta Tag" Lighthouse audit.
+description: Reference documentation for the "Has A <meta name="viewport"> Tag With width Or initial-scale" Lighthouse audit.
```
Comment (petele) [inline src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md]: DevSite misbehaves when there are HTML tags in the description, I'd suggest either encoding, or changing to something like "Has a meta viewport Tag..."

File: src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-{# wf_updated_on: 2016-10-04 #}
+{# wf_updated_on: 2017-04-18 #}
 {# wf_published_on: 2016-10-04 #}
 
-# HTML Has a Viewport Meta Tag {: .page-title }
+# Has A <meta name="viewport"> Tag With width Or initial-scale {: .page-title }
```
Comment (petele) [inline src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md]: DevSite misbehaves when there are HTML tags in the document title, I'd recommend same as document description.

File: src/content/en/tools/lighthouse/audits/notifications-on-load.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
-description: Reference documentation for the "Page Does Not Automatically Request Notification Permissions On Page Load" Lighthouse audit.
+description: Reference documentation for the "Avoids Requesting The Notification Permission On Page Load" Lighthouse audit.
 
 {# wf_updated_on: 2016-12-05 #}
```
Comment (petele) [inline src/content/en/tools/lighthouse/audits/notifications-on-load.md]: don't forget to update `wf_updated_on`

File: src/content/en/tools/lighthouse/audits/required-aria-attributes.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 project_path: /web/_project.yaml
 book_path: /web/tools/_book.yaml
-description: Reference documentation for the "Elements With ARIA Roles Have Required Attributes" Lighthouse audit.
+description: Reference documentation for the "Elements With ARIA Roles Have The Required aria-* Attributes" Lighthouse audit.
 
 {# wf_updated_on: 2017-01-17 #}
```
Comment (petele) [inline src/content/en/tools/lighthouse/audits/required-aria-attributes.md]: don't forget to update `wf_updated_on`

File: src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/lighthouse/audits/has-viewport-meta-tag.md]: I encoded. I know it looks funky, but the goal is for the titles to match exactly for SEO

**Comment by kaycebasques** [comment]
I'm fixing the ToC... will re-open shortly

**Comment by kaycebasques** [comment]
Argh it's still throwing the "top-level el cannot contain HTML" even though I encoded it... I'll just hack that one to something else...

**Pull request merged**